examples/experimental/i3s-17-and-debug/tile-debug.js

```diff
@@ -39,6 +39,98 @@ export function getTileDebugInfo(tileHeader) {
   };
 }
 
+export function validateTile(tile) {
+  const tileWarnings = [];
+
+  checkBoundingVolumes(tile, tileWarnings);
+  checkLOD(tile, tileWarnings);
+
+  return tileWarnings;
+}
+
+export function isTileGeometryInsideBoundingVolume(tile) {
+  const boundingType = getBoundingType(tile);
+  const positions = tile.content.attributes.positions.value;
+  let boundingVolume = null;
+
+  switch (boundingType) {
+    case OBB: {
+      boundingVolume = createBoundingBoxFromTileObb(tile.header.obb);
```

Comment:
I wonder if the I3SLoader shouldn't create these when it generates a "normalized" tile.